src/hepunits/units/units.py

```diff
@@ -63,91 +63,91 @@
 
 from math import pi
 
-from .prefixes import *
+from . import prefixes as _pre
```

Comment:
This is what you refer to as leaky abstractions? Just for my understanding.

Reply:
The problem is you have circular `import *`'s. MyPy can handle circular imports or starred imports, but not both. `hepunits/__init__.py` imports the subpackages, but subpackages import from a different subpackage, which imports the parent package, which causes a circle. Python is smart enough to break the loop, and MyPy is too, but not if there are too many `import *`'s. The `import *`'s should only be outward facing; internally you should not be `import *`'ing from one subpackage to another.

Reply:
OK, thanks. Makes sense.